app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/navigation/PaymentsGraph.kt

```diff
@@ -91,13 +96,33 @@ fun NavGraphBuilder.paymentsGraph(
       )
     }
 
+    composable<PaymentsDestinations.Forever>(
+      deepLinks = listOf(
+        navDeepLink { uriPattern = hedvigDeepLinkContainer.forever },
+      ),
+      enterTransition = { MotionDefaults.fadeThroughEnter },
+      exitTransition = { MotionDefaults.fadeThroughExit },
```

Comment:
```suggestion
```
These can be removed as well, so that way the normal x-axis animation can play here, like any other screen

These fade transitions are specifically made for those top level destinations only.

It is a bit simple how we've set this all up, but due to lack of more specific design guidance on this we decided on this approach. Fade for top level, x axis for everything else.

So "top-level" here https://m3.material.io/styles/motion/transitions/transition-patterns#8327d206-2a7d-423a-abf0-2be86130535b

And forward and backward here https://m3.material.io/styles/motion/transitions/transition-patterns#df9c7d76-1454-47f3-ad1c-268a31f58bad for the x-axis that I am talking about